pkgs/applications/video/lbry/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv
+, dpkg
+, fetchurl
+, at-spi2-core
+, atomEnv
+, ffmpeg
+, gtk3
+, libdrm
+, libpulseaudio
+, libuuid
+, makeWrapper
+, mesa
+, xdg_utils
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lbry-desktop";
+  version = "0.48.2";
+  src = fetchurl {
+    url = "https://github.com/lbryio/lbry-desktop/releases/download/v${version}/LBRY_${version}.deb";
+    sha256 =  "0kqyiv9lfck9zzhiqpbvmlbyk09hajcaa2bpk1i3w03yk548an6z";
+  };
+
+  rpath = stdenv.lib.makeLibraryPath [
+    at-spi2-core
+    libdrm
+    libpulseaudio
+    libuuid
+    mesa
+    xorg.libXScrnSaver
+    xorg.libxcb
+  ] + ":${atomEnv.libPath}";
+
+  buildInputs = [
+    ffmpeg
+    gtk3
+  ];
+
+  nativeBuildInputs = [ dpkg makeWrapper ];
+
+  dontUnpack = true;
+  dontBuild = true;
+  dontPatchELF = true;
+
+  installPhase = ''
+      dpkg --fsys-tarfile $src | tar --extract
+      rm -rf usr/share/lintian
+
+      mkdir -p $out/bin
+      mv usr/* $out
+      mv opt/LBRY $out/
+
+      # Otherwise it looks "suspicious"
+      chmod -R g-w $out
+
+      for file in $(find $out -type f \( -perm /0111 -o -name \*.so\* \) ); do
+        patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" "$file" || true
```

Comment:
Can you please remove the ``|| true`` that we notice errors.

Reply:
I did that because I was going off the Slack expression, which does that. Guess there are reasons that app needs to eat errors that this one doesn't.

Reply:
Eh, now there's an error from patchelf: "cannot find section '.interp'. The input file is most likely statically linked"

Reply:
I've tried removing this section and letting the fixupPhase do it's default thing, but that causes problems too. I'm going to leave this as-is because it actually works, and there's precedent from the Slack package as well.